DeviceAdministration/Web/WebApiControllers/TelemetryApiController.cs

```diff
@@ -250,11 +250,18 @@ public class TelemetryApiController : WebApiControllerBase
                         historyItems.AddRange(data);
 
                         List<dynamic> devices = await LoadAllDevicesAsync();
-                        devices.ForEach(device => TypeMapper.Get().map<DeviceND>(device));
+                        List<DeviceND> resultND = new List<DeviceND>();
+
+
+                        foreach(var device in devices)
+                        {
+                            DeviceND d2 = TypeMapper.Get().map<DeviceND>(device);
+                            resultND.Add(d2);
+                        }
```

Comment:
I think you can do this
List<DeviceND> resultND = TypeMapper.Get().Map<List<DeviceND>>(devices)